make docker-otelsvc fails

GOOS=linux /Library/Developer/CommandLineTools/usr/bin/make otelsvc
GO111MODULE=on CGO_ENABLED=0 go build -o ./bin/otelsvc_linux -ldflags "-X github.com/open-telemetry/opentelemetry-service/internal/version.GitHash=3aac7c1 " ./cmd/otelsvc
cp ./bin/ocotelsvc_linux ./cmd/ocotelsvc/
cp: directory ./cmd/ocotelsvc does not exist
make[1]: *** [docker-component] Error 1
make: *** [docker-otelsvc] Error 2

--- a/Makefile
+++ b/Makefile
@@ -92,9 +92,9 @@ otelsvc:
 .PHONY: docker-component # Not intended to be used directly
 docker-component: check-component
 	GOOS=linux $(MAKE) $(COMPONENT)
-	cp ./bin/oc$(COMPONENT)_linux ./cmd/oc$(COMPONENT)/
-	docker build -t oc$(COMPONENT) ./cmd/oc$(COMPONENT)/
-	rm ./cmd/oc$(COMPONENT)/oc$(COMPONENT)_linux
+	cp ./bin/$(COMPONENT)_linux ./cmd/$(COMPONENT)/
+	docker build -t $(COMPONENT) ./cmd/$(COMPONENT)/
+	rm ./cmd/$(COMPONENT)/$(COMPONENT)_linux
 
 .PHONY: check-component
 check-component:

